plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -62,15 +63,15 @@ def __init__(self) -> None:
         if Platform.isLinux() and not default_engine_location:
             if not os.getenv("PATH"):
                 raise OSError("There is something wrong with your Linux installation.")
-            for pathdir in os.getenv("PATH").split(os.pathsep):
+            for pathdir in cast(str, os.getenv("PATH")).split(os.pathsep):
```

Comment:
if os.getenv("PATH") returns None for some reason, the statement will break.

Reply:
This is a case where MyPy is not smart enough. Already check above if os.getenv("PATH") is none in the line above and raise error accordingly. Won't fix.